geth/api/backend.go

```diff
@@ -27,14 +27,15 @@ type StatusBackend struct {
 func NewStatusBackend() *StatusBackend {
 	defer log.Info("Status backend initialized")
 
-	nodeManager := node.NewNodeManager()
+	nodeManager := proxy.NewPRCRouter(node.NewNodeManager())
 	accountManager := node.NewAccountManager(nodeManager)
+
 	return &StatusBackend{
 		nodeManager:    nodeManager,
 		accountManager: accountManager,
-		txQueueManager: node.NewTxQueueManager(nodeManager, accountManager),
 		jailManager:    jail.New(nodeManager),
 		rpcManager:     node.NewRPCManager(nodeManager),
```

Comment:
`statuc-react` will be calling RPC methods with `CallRPC`. This signal calls `StatusBackend.rpcManager.RPCServer()` in the end and requests are handled with the current go-ethereum RPC Server ignoring the new upstream config and this needs fixing.